### PR TITLE
An 463 anchor table update with the DBT test

### PR DESCRIPTION
**Pull request by XYGUAN**

### CODE REVIEW

File: models/terra/anchor/anchor__collateral.sql [hunk 1]
```diff
@@ -139,8 +139,8 @@ SELECT
   ) AS amount,
   amount * price AS amount_usd,
   msg_value :contract :: STRING AS currency,
-  msg_value :execute_msg :send :contract :: STRING AS contract_address,
-  l.address AS contract_label
+  COALESCE(msg_value :execute_msg :send :contract :: STRING, '') AS contract_address,
+  COALESCE(l.address, '') AS contract_label
```
Comment (jkallogjeri) [inline models/terra/anchor/anchor__collateral.sql:143]: change this to l.address_name

File: models/terra/anchor/anchor__redeem.sql [hunk 2]
```diff
@@ -49,8 +49,8 @@ SELECT
   ) AS amount,
   amount * price AS amount_usd,
   msg_value :contract :: STRING AS currency,
-  msg_value :execute_msg :send :contract :: STRING AS contract_address,
-  l.address AS contract_label
+  COALESCE(msg_value :execute_msg :send :contract :: STRING, '') AS contract_address,
+  COALESCE(l.address, '') AS contract_label
```
Comment (jkallogjeri) [inline models/terra/anchor/anchor__redeem.sql:53]: l.address_name

File: models/terra/anchor/anchor__repay.sql [hunk 3]
```diff
@@ -49,8 +49,8 @@ SELECT
   ) AS amount,
   amount * price AS amount_usd,
   msg_value :coins [0] :denom :: STRING AS currency,
-  msg_value :contract :: STRING AS contract_address,
-  l.address AS contract_label
+  COALESCE(msg_value :contract :: STRING, '') AS contract_address,
+  COALESCE(l.address, '') AS contract_label
```
Comment (jkallogjeri) [inline models/terra/anchor/anchor__repay.sql:53]: l.address_name

Comment (XYGUAN) [reply models/terra/anchor/anchor__repay.sql]: Yup, I updated all the tables address name for the contract label

**Review by jkallogjeri — APPROVED** [review]

**Pull request merged**